### PR TITLE
Naming like organization

### PR DESCRIPTION
**Pull request by hannesa2**
* package like organize name
* filename `js2d` where necessary

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -25,8 +25,8 @@ gradlePlugin {
         jsonschema2dataclassPlugin {
             id = 'org.jsonschema2dataclass'
             version = getTag()
-            implementationClass = 'com.github.eirnym.js2p.JsonSchemaPlugin'
-            displayName = 'Extended Gradle JsonSchema2Pojo Plugin'
+            implementationClass = 'com.github.js2d.JsonSchemaPlugin'
+            displayName = 'Extended Gradle JsonSchema2DataClass Plugin'
```
Comment (eirnym) [inline build.gradle:29]: Revert displayName

**Review by eirnym — CHANGES_REQUESTED** [review]
Only displayName should be the same as it was. Rest of changes is good

File: build.gradle (hunk 1 above)
Comment (hannesa2) [reply build.gradle]: I did it, but why this old name ?

Comment (eirnym) [reply build.gradle]: _Currently_ this is a gradle plugin for JsonSchema2Pojo library. Probably this should be stated better.

**Review by eirnym — APPROVED** [review]

**Comment by eirnym** [comment]
@hannesa2 you're free to squash and merge.

**Comment by hannesa2** [comment]
Sorry, I forgot to squash

**Pull request merged**